registrations/tests/test_registration_get.py

```diff
@@ -143,12 +145,53 @@ def test_admin_can_get_registration_not_created_by_self(
 @pytest.mark.django_db
 def test_admin_user_can_see_registration_user_accesses(registration, user_api_client):
     RegistrationUserAccess.objects.create(registration=registration)
+    RegistrationUserAccess.objects.create(registration=registration, email=test_email)
```

Comment:
Could use `RegistrationUserAccessFactory` instead of `RegistrationUserAccess.objects.create`. Applies to everything in this file. Honestly, with simple test cases like this, doesn't really matter that much, but personally, I forget the existence of factories every now and then. Seeing them used in other tests reminds me of their existence and encourages me to use them in my own tests as well.